lib/fluent/event.rb

```diff
@@ -144,16 +160,20 @@ def each(&block)
 
   class MessagePackEventStream < EventStream
     # Keep cached_unpacker argument for existence plugins
-    def initialize(data, cached_unpacker = nil)
+    def initialize(data, records = 0, cached_unpacker = nil)
```

Comment:
Some 3rd party plugins use `MessagePackEventStream.new(data, @cached_unpadker)` in their code.
Should we break these code?